app/src/main/java/chat/rocket/android/contacts/ui/ContactsFragment.kt

```diff
@@ -284,7 +311,12 @@ class ContactsFragment : Fragment(), ContactsView {
                     filteredContactArrayList.add(contact)
                 }
             }
-            setupFrameLayout(filteredContactArrayList)
+            launch(UI) {
```

Comment:
@kb0304 my app is crashing in the emulator every time i do a search from the contacts page right now.... 

java.lang.IllegalStateException: Cannot access database on the main thread since it may potentially lock the UI for a long period of time.
        at androidx.room.RoomDatabase.assertNotMainThread(RoomDatabase.java:209)
        at androidx.room.RoomDatabase.query(RoomDatabase.java:237)
        at chat.rocket.android.db.UserDao_Impl.getUser(UserDao_Impl.java:256)
        at chat.rocket.android.server.domain.GetCurrentUserInteractor.get(GetCurrentUserInteractor.kt:13)
        at chat.rocket.android.chatrooms.adapter.RoomUiModelMapper$currentUser$2.invoke(RoomUiModelMapper.kt:32)
        at chat.rocket.android.chatrooms.adapter.RoomUiModelMapper$currentUser$2.invoke(RoomUiModelMapper.kt:25)
        at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
        at chat.rocket.android.chatrooms.adapter.RoomUiModelMapper.getCurrentUser(Unknown Source:25)
        at chat.rocket.android.chatrooms.adapter.RoomUiModelMapper.isChannelWritable(RoomUiModelMapper.kt:155)
        at chat.rocket.android.chatrooms.adapter.RoomUiModelMapper.mapRoom(RoomUiModelMapper.kt:100)
        at chat.rocket.android.chatrooms.adapter.RoomUiModelMapper.map(RoomUiModelMapper.kt:59)
        at chat.rocket.android.contacts.ui.ContactsFragment$queryContacts$1.doResume(ContactsFragment.kt:315)
        at kotlin.coroutines.experimental.jvm.internal.CoroutineImpl.resume(CoroutineImpl.kt:42)

Reply:
> If there are no contacts showing in the results, should we remove the INVITE CONTACTS header?

Done